internal/xdg/xdg.go

```diff
@@ -1,5 +1,5 @@
 /*
-Package config manages autokitteh's configuration and data directories,
+This manages autokitteh's configuration and data directories,
```

Comment:
Package documentation should begin with "Package <Name>..." - see https://tip.golang.org/doc/comment. Bug given the package name change, it should be "Package XDG manages..."

Reply:
done